doc/techref/projections.md

```diff
@@ -16,34 +16,34 @@ The table below shows the projection codes for the 31 GMT map projections:
-| **W**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_mollweide` |
-| **X***width*[**l**\|**p***exp*\|**T**\|**t**][/*height*[**l**\|**p***exp*\|**T**\|**t**]][**d**] | Cartesian {doc}`linear </projections/nongeo/cartesian_linear>`, {doc}`logarithmic </projections/nongeo/cartesian_logarithmic>`, {doc}`power </projections/nongeo/cartesian_power>`, and time |
-| **Y**{{ lon0 }}/{{ lat0 }}/*width*                          | {doc}`/projections/cyl/cyl_equal_area` |
+| **A**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | [](/projections/azim/azim_lambert.rst) |
+| **B**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | [](/projections/conic/conic_albers.rst) |
+| **C**{{ lon0 }}/{{ lat0 }}/*width*                          | [](/projections/cyl/cyl_cassini.rst) |
+| **Cyl_stere**/[{{ lon0 }}/[{{ lat0 }}/]]*width*             | [](/projections/cyl/cyl_stereographic.rst) |
+| **D**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | [](/projections/conic/conic_equidistant.rst) |
+| **E**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | [](/projections/azim/azim_equidistant.rst) |
+| **F**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | [](/projections/azim/azim_gnomonic.rst) |
+| **G**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | [](/projections/azim/azim_orthographic.rst) |
+| **G**{{ lon0 }}/{{ lat0 }}/*width*[**+a***azimuth*][**+t***tilt*][**+v***vwidth*/*vheight*][**+w***twist*][**+z***altitude*] | [](/projections/azim/azim_general_perspective.rst) |
+| **H**[{{ lon0 }}/]*width*                                   | [](/projections/misc/misc_hammer.rst) |
+| **I**[{{ lon0 }}/]*width*                                   | [](/projections/misc/misc_sinusoidal.rst) |
+| **J**[{{ lon0 }}/]*width*                                   | [](/projections/cyl/cyl_miller.rst) |
+| **Kf**[{{ lon0 }}/]*width*                                  | [](/projections/misc/misc_eckertIV.rst) |
+| **Ks**[{{ lon0 }}/]*width*                                  | [](/projections/misc/misc_eckertVI.rst) |
+| **L**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | [](/projections/conic/conic_lambert.rst) |
+| **M**[{{ lon0 }}/[{{ lat0 }}/]]*width*                      | [](/projections/cyl/cyl_mercator.rst) |
+| **N**[{{ lon0 }}/]*width*                                   | [](/projections/misc/misc_robinson.rst) |
+| **Oa**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]             | [Oblique Mercator projection: 1. origin and azimuth](/projections/cyl/cyl_oblique_mercator.rst) |
+| **Ob**{{ lon0 }}/{{ lat0 }}/{{ lon1 }}/{{ lat1 }}/*width*[**+v**] | [Oblique Mercator projection: 2. two points](/projections/cyl/cyl_oblique_mercator.rst) |
+| **Oc**{{ lon0 }}/{{ lat0 }}/{{ lonp }}/{{ latp }}/*width*[**+v**] | [Oblique Mercator projection: 3. origin and projection pole](/projections/cyl/cyl_oblique_mercator.rst) |
```

Comment:
Before we only included the different options of the Oblique Merator projection in the links, simliar to the Cartesian projection.

```suggestion
| **Oa**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]             | Oblique Mercator projection: [1. origin and azimuth](/projections/cyl/cyl_oblique_mercator.rst) |
| **Ob**{{ lon0 }}/{{ lat0 }}/{{ lon1 }}/{{ lat1 }}/*width*[**+v**] | Oblique Mercator projection: [2. two points](/projections/cyl/cyl_oblique_mercator.rst) |
| **Oc**{{ lon0 }}/{{ lat0 }}/{{ lonp }}/{{ latp }}/*width*[**+v**] | Oblique Mercator projection: [3. origin and projection pole](/projections/cyl/cyl_oblique_mercator.rst) |
```